lib/internal/event_target.js

```diff
@@ -15,11 +15,12 @@ const {
     ERR_INVALID_ARG_TYPE,
     ERR_EVENT_RECURSION,
     ERR_OUT_OF_RANGE,
+    ERR_INVALID_THIS,
   }
 } = require('internal/errors');
 
 const perf_hooks = require('perf_hooks');
-const { customInspectSymbol } = require('internal/util');
+const { customInspectSymbol, kIsEventTarget } = require('internal/util');
```

Comment:
Nit: why define `kIsEventTarget` in a separate file? It could just be included in this file?

Reply:
From looking around the node codebase, it seemed like some calls to `SymbolFor` happened there. [On another glance](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/nodejs/node%24+SymbolFor+lang:js+-file:deps+-file:test+-file:node_modules&patternType=literal#1), it looks like there's more usage inside files, so np, I'll move it in here.